types/firestore.d.ts

```diff
@@ -1519,8 +1562,8 @@ declare namespace FirebaseFirestore {
     where(
       fieldPath: string | FieldPath,
       opStr: WhereFilterOp,
-      value: any
-    ): Query<T>;
+      value: unknown
```

Comment:
Was it necessary to change `any` to `unknown`? If no, please revert it back to `any` in order to keep this PR focused.